STACpopulator/extensions/datacube.py

```diff
@@ -202,6 +207,25 @@ def variables(self) -> dict[str, Variable]:
             )
         return variables
 
+    def _infer_variable_units_description(self, name, attrs):
+        """Try to infer the units and description of some simple coordinate variables."""
+        if name == "time_bnds":
+            related_variable = "time"
+            attrs["description"] = "bounds for the time coordinate"
+        elif name == "lat_bnds":
+            related_variable = "lat"
+            attrs["description"] = "bounds for the latitude coordinate"
+        elif name == "lon_bnds":
+            related_variable = "lon"
+            attrs["description"] = "bounds for the longitude coordinate"
+        else:
+            return
```

Comment:
I suspect you'd be able to harden the bound variables inference using cf-xarray. It looks for the bounds attributes of variables. See https://cfconventions.org/cf-conventions/cf-conventions.html#cell-boundaries

Reply:
https://cf-xarray.readthedocs.io/en/latest/generated/xarray.Dataset.cf.bounds.html#xarray.Dataset.cf.bounds

Reply:
Now I start to remember. We didn't want to add cf-xarray as a dependency, right? So we just copied some of the logic here. Either we bring more of the cf logic into the repo, or add is as a dependency ? No strong opinion.

Reply:
Ok, then we can keep it cf-array free for now. I think this will be the path of least resistance, and time is a bit of an issue at the moment.

Reply:
@David now all the bounds variables are shown as auxiliary